addons/revolutionaries/CfgVehicles.hpp

```diff
@@ -0,0 +1,106 @@
+class CfgVehicles {
+    // Base Classes
+    class TACU_Main_U_OPFOR_Soldier_Base;
+	class TACU_Main_U_INDEP_Soldier_Base;
+  #include "CfgVehicles_Tanoan.hpp"
+  #include "CfgVehicles_Russian.hpp"
+  #include "CfgVehicles_TanoanHD.hpp"
+  #include "CfgVehicles_RussianHD.hpp"
+  #include "CfgVehicles_Vehicles.hpp"
+  
+ //Backpacks
+    class B_FieldPack_blk;
+	    class TACU_Revolutionaries_Backpack_Mk14: B_FieldPack_blk {
```

Comment:
Formatting on this one.
```suggestion
    class TACU_Revolutionaries_Backpack_Mk14: B_FieldPack_blk {
```